docs/source/extensions/plugins.rst

```diff
@@ -70,7 +70,7 @@ Expert users may choose to extend an existing plugin by overriding its methods .
                 device_ids=...,
             )
 
-or by subclassing the base classes :class:`~pytorch_lightning.plugins.training_type.Strategy` or
+or by subclassing the base classes :class:`~pytorch_lightning.plugins.training_type.training_type_plugin` or
```

Comment:
I suppose this should be kept as is.
```suggestion
or by subclassing the base classes :class:`~pytorch_lightning.plugins.training_type.Strategy` or
```

Reply:
not yet. this was accidentally changed in another PR. The files have not been renamed yet.